src/pages/all-about-carbon/who-uses-carbon.mdx

```diff
@@ -134,6 +134,7 @@ Here are some ways developers can begin engaging with Carbon.
 
 - Check out a tutorial in your framework of choice
   ([React using Next.js](/developing/react-tutorial/overview/),
+  ([Web components using Vaniall.js/HTML](/developing/web-components-tutorial/overview/),
```

Comment:
Looking good. We'll need a dev or two go through the tutorial to double-check it.

Reply:
Definitely, needs an update following yesterdays release (PRs mentioned in tutorial).

It would benefit massively from Grid and Column being web components. I may be able to contribute a partial solution in the next couple of weeks.